nomad/structs/structs.go

```diff
@@ -4457,6 +4457,19 @@ type Job struct {
 	ModifyIndex uint64
 	// JobModifyIndex is the index at which the job *specification* last changed
 	JobModifyIndex uint64
+
+	// Links and Description fields for the Web UI
+	Ui *JobUIConfig
+}
+
+type JobUIConfig struct {
+	Description string
+	Links       []JobUILink
+}
```

Comment:
This will need a `Copy` and `Equal` method, and have it called from the `Copy`/`Equal` method of `Job`.